src/Components/Main.tsx

```diff
@@ -26,48 +19,57 @@ export interface IProps {
   visible?: boolean;
   reduxActions: ReduxAction[];
   rnRequests: RNRequest[];
+  nRequests: NRequest[];
   clearAll: Function;
   maxRequests?: number;
 }
 
-let reduxList: ReduxAction[] = [];
-let rnList: RNRequest[] = [];
-// let nList: NRequest[] = [];
+const _positionFlatList = 0;
 
 export const Main = (props: IProps) => {
+  const theme = useContext(ThemeContext);
+  const refFlatList = useRef<FlatList<any>>(null);
+  const [flatListLastOffset, setFlatListLastOffset] = useState<number>(0);
```

Comment:
it's the fix related to flatlist position? is it fully OK to use in library?

Reply:
It works well if you add just one element in the list. However, if there are more calls in one time, the position will be lost. It's only for Android.